mitodl/mit-open
# certifcation should be a boolean

# What are the relevant tickets?
closes https://github.com/mitodl/mit-open/issues/248

# Description (What does it do?)
- Changes the "certification" parameter to a boolean similar to "professional"
- Fixes a bug that cause mitx learning resources to not be marked as having certificates

# How can this be tested?
Run docker-compose run web ./manage.py recreate_index --all
http://localhost:8063/api/v1/learning_resources_search/?certification=true
and
http://localhost:8063/api/v1/learning_resources_search/?certification=false 
should work to filter by certification

Certification should be a boolean in the results
